lib/iris/tests/integration/test_FieldsFileVariant.py

```diff
@@ -24,12 +24,22 @@
 
 import shutil
 import tempfile
+import unittest
 
 import numpy as np
 
 from iris.experimental.um import (Field, Field2, Field3, FieldsFileVariant,
                                   FixedLengthHeader)
 
+try:
+    import mo_unpack
```

Comment:
What is `mo_unpack`? Should this be `mo_pack` instead? I'm getting a bunch of skipped tests saying I need `mo_unpack`...